test/framework/kogitoapp.go

```diff
@@ -137,6 +139,34 @@ func cliDeployExample(namespace string, kogitoAppDeployment KogitoAppDeployment)
 	return err
 }
 
+// DeployExampleServiceFromYaml deploy an example service from YAML file (example is located in deploy/examples folder)
+func DeployExampleServiceFromYaml(namespace, exampleServiceYaml string) error {
+	box := packr.New("examples", boxExamplesPath)
+	yamlContent, err := box.FindString(exampleServiceYaml)
+	if err != nil {
+		return fmt.Errorf("Error reading file %s: %v ", exampleServiceYaml, err)
+	}
+
+	// status.conditions cannot be omitted as conditions is an embedded struct, therefore it doesn't have default value
+	// As a workaround empty conditions have to be provided when creating Kogito CR.
+	kogitoApp := &v1alpha1.KogitoApp{
```

Comment:
I'd say you guys could have a util function to create those empty instances for you

Reply:
Or with some kind of parameter to customize the instance. I remember seeing this kind of code more than once.

Reply:
Adjusted the implementation to use available KogitoApp stub.